AsyncSwift/Observed/EventDetailView+Observed.swift

```diff
@@ -0,0 +1,50 @@
+//
+//  EventDetailView+Observed.swift
+//  AsyncSwift
+//
+//  Created by Kim Insub on 2022/09/09.
+//
+
+import Foundation
+import EventKit
+
+extension EventDetailView {
+    final class Observed: ObservableObject {
+
+        @Published var showSheet = false
+        @Published var showingAlert = false
+
+        let eventStore : EKEventStore = EKEventStore()
```

Comment:
```suggestion
        private let eventStore = EKEventStore()
```

Private 설정과 타입 생략이 되면 좋을 것 같아요 ~

Reply:
확인했습니다 👍